<!DOCTYPE html>

<html xmlns="http://www.w3.org/1999/xhtml">

<head>

<meta charset="utf-8">
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta name="generator" content="pandoc" />


<meta name="author" content="Paul Oldham" />

<meta name="date" content="2017-01-10" />

<title>Introduction</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/bootstrap.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-1.1/default.css" rel="stylesheet" />
<script src="site_libs/highlightjs-1.1/highlight.js"></script>

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs && document.readyState && document.readyState === "complete") {
   window.setTimeout(function() {
      hljs.initHighlighting();
   }, 0);
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>


</head>

<body>

<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
  height: auto;
}
.tabbed-pane {
  padding-top: 12px;
}
button.code-folding-btn:focus {
  outline: none;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}

.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>


<div class="container-fluid main-container">

<!-- tabsets -->
<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});
</script>

<!-- code folding -->






<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Handbook on ABS Monitoring</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Introduction</a>
</li>
<li>
  <a href="gettingstarted.html">Getting Started</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Working with Taxonomic Data
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="gbif.html">Accessing GBIF</a>
    </li>
    <li>
      <a href="mapgbif.html">Mapping GBIF Data</a>
    </li>
  </ul>
</li>
<li>
  <a href="crossref.html">Accessing Scientific Literature</a>
</li>
<li>
  <a href="geonames.html">Mapping with Geonames</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Introduction</h1>
<h4 class="author"><em>Paul Oldham</em></h4>
<h4 class="date"><em>10 January 2017</em></h4>

</div>


<p>This Handbook focuses on methods and techniques for monitoring the use of genetic resources and traditional knowledge under the <a href="https://www.cbd.int/abs/about/">Nagoya Protocol</a> of the <a href="https://www.cbd.int/">Convention on Biological Diversity</a>. The aim of the Handbook is to assist governments with identifying practical tools and methods for implementing monitoring under the Nagoya Protocol as a precondition for building trust in international relationships involving exchanges of genetic resources and associated traditional knowledge.</p>
<p>The Handbook is in a very early stage of development. Drawing on the publication model in data science, draft articles will be published as they become available and will then be corrected, revised and expanded. Comments are welcome and can be made using Github issues at the project repository <a href="https://github.com/poldham/abs">here</a>. This is an open source project and all data is available in the <a href="https://github.com/poldham/abs">Github project repository</a>. Please see the <a href="https://github.com/poldham/abs/contributing.html">Contributing</a> page for more information.</p>
<p>The Handbook explores software tools and methods for tracking and monitoring genetic resources and traditional knowledge. The Handbook is intended to support the development and implementation of an <a href="http://abspermits.net/index.html">online permit and monitoring system</a> in Kenya and other partner countries. The Handbook is being developed with support from the <a href="http://www.abs-initiative.info/">ABS Initiative</a> and contributions from partner countries interested in developing monitoring capacity.</p>
<p><a href="https://www.cbd.int/abs/text/articles/default.shtml?sec=abs-17">Article 17</a> of the Nagoya Protocol calls upon countries to develop cost effective tools to enhance transparency about the utilisation of genetic resources and associated traditional knowledge. Countries will vary in their monitoring needs depending on whether they are focusing on their roles as either users or providers of genetic resources and resources. As a starting point in considering monitoring it is useful to think about the basic question of what can be monitored.</p>
<div id="what-can-be-monitored" class="section level3">
<h3>What can be monitored</h3>
<p>The main areas that can be monitored are:</p>
<ol style="list-style-type: decimal">
<li>Publications in the scientific literature</li>
<li>Patent publications (as a reflection of R&amp;D investments)</li>
<li>Digital sequence information (DNA and amino acid sequences)</li>
<li>Product information;</li>
<li>Administrative information such as research permits, international certificates of compliance (IRCCs), and declarations of due diligence.</li>
</ol>
<p>Other areas that might be considered include imports and exports of materials. However, as will become apparent, key issues to think about when designing monitoring measures are:</p>
<ol style="list-style-type: lower-alpha">
<li>Accessibility, that is how easy is it to access information;</li>
<li>Complexity, in terms of how complex or messy is the data, and;</li>
<li>Scale, in terms of whether we are dealing with hundreds, thousands or millions of things to be monitored.</li>
</ol>
<p>Some of these areas are easier to access and work with than others.</p>
<p>This Handbook explores and provides examples of the use of software and database tools that can be used by governments to monitor research and patent activity under the Nagoya Protocol. The main focus of the Handbook is on the growing availability of free software tools, notably web services or Application Programming Interfaces, that can be used to freely access scientific, patent and sequence information.</p>
<p>The Handbook focuses on the strengths and limitations of data sources for monitoring in the development of scientific and patent landscapes for access and benefit-sharing. Landscapes for scientific research and patent activity seek to identify:</p>
<ol style="list-style-type: lower-alpha">
<li>Trends in activity over time</li>
<li>The biological organisms involved</li>
<li>The organisations involved</li>
<li>The researchers involved</li>
<li>Sectors, technology areas and markets involving research, and/or research and development, utilizing genetic resources and associated traditional knowledge</li>
<li>Evidence of revenue generated by products or licensing</li>
</ol>
<p>In practice, governments seeking to engage in monitoring of access and benefit sharing agreements under the Nagoya Protocol are likely to utilize a range of commercial and free software and database tools. At present there is a limited understanding of the strengths and limitations of these tools for monitoring under the Nagoya Protocol.</p>
<p>This Handbook seeks to advance understanding of the strengths and limitations of these tools using data about Kenya as the main example. The Handbook forms part of a wider family of projects involving the Bahamas, Kenya and India to facilitate monitoring of access and benefit-sharing involving genetic resources and traditional knowledge. It is expected that the Handbook will be useful to other projects and countries as a starting point to support practical work on monitoring under the Nagoya Protocol.</p>
<p>At a later stage the Handbook will demonstrate the use of commercial tools, such as Web of Science and Thomson Innovation. However, as a starting point the Handbook will focus as far as possible on pursuing cost-effectiveness by focusing on the potential of free tools. The landscape for free software tools and databases has changed dramatically in recent years. Specifically:</p>
<ul>
<li>a growing number of countries are promoting open access policies for the results of scientific research,</li>
<li>databases are increasingly providing web services (Application Programming Interfaces or APIs) that provide free access to the contents, or metadata about scientific and related subjects.</li>
<li>A range of software tools and programming languages are focusing on access to data as a precondition for the rise of data science.</li>
</ul>
<p>An important issue when considering the use of free software tools such as APIs is that they will generally require engagement with programming languages such as JavaScript, Ruby, Python or R (among others) and will typically be linked with database software. Governments seeking to implement monitoring for access and benefit sharing should anticipate from the start that this is a technical exercise that will require investment in technical expertise, either in house or through contracted services. That is, the main costs in implementing monitoring for access and benefit-sharing will be in human knowledge and technical capacity.</p>
<p>This report demonstrates the use of free tools using the statistical programming language R. R was selected for this task for a variety of reasons.</p>
<ol style="list-style-type: decimal">
<li>Scientific and patent landscaping involves the generation of statistical information that, over the longer term, can be linked with modelling and forecasting.</li>
<li>R is supported by over 10,000 packages for different tasks including mapping and an ever increasing number of packages for free and easy access to web services and data services.</li>
<li>R is supported by easy to use and powerful user interfaces, notably <a href="https://www.rstudio.com/">RStudio</a>, a huge ecosystem of users and extensive online training materials for different tasks.</li>
<li>R is widely used in biochemistry and genomics research through a specialised set of <code>bioconductor</code> packages.</li>
<li>The R user community is associated with a drive towards <code>reproducible research</code>, whereby sufficient information is provided that other researchers can reproduce the results. Reproducible methods are important in quality control for monitoring in access and benefit-sharing, for disseminating methods for use by the wider ABS community and for improving on methods over time</li>
</ol>
<p>While this Handbook will use R, it is important to note that the choices made by governments on monitoring tools will in part depend on the available expertise. For example, Python is a very popular general programming language and most of the tasks that can be performed in R can readily be performed in Python. If there is a much greater expertise in the use of Python or another language it may make sense to base monitoring tools on that language with due regard to the promotion of transparent and reproducible methods.</p>
<p>As we will see below, it is possible to achieve a great deal using free tools. However, the report also exposes some significant limitations with existing tools for access and benefit-sharing. These can be briefly summarised as possible.</p>
<ol style="list-style-type: decimal">
<li>International scientific publications are distributed in a myriad of pay walled (pay per view silos). Free tools such as CrossRef and PubMed provide access to limited metadata about publications that can be used in ABS monitoring. In addition, commercial tools such as Thomson Reuters Web of Science provide access to a great range of meta data. However, at present access to data is a significant obstacle to monitoring as we will see in the case of CrossRef.</li>
<li>Access to patent data using free patent databases is typically limited to metadata (front pages) and the ability to search the full text (where genetic resources are typically referenced) are presently limited. This situation is changing through full text search facilities using the Lens free patent database and WIPO Patentscope. In addition, the USPTO provides for bulk downloads of its entire patent collection. As such the situation is improving but data access in the form required for effective monitoring is presently limited.</li>
<li>Issues around data access relate to the ability to engage in monitoring at scale. Taxonomic information about genetic resources recorded in international databases such as the <a href="http://www.gbif.org/">Global Biodiversity Information Facility</a> are typically in the order of 10s of thousands of species. This presents challenges in terms of searching millions of documents and, where access to the data is possible, require advanced text mining methods. We are not there yet, but the situation is improving.</li>
</ol>
<p>In light of these challenges in terms of data access and the ability to operate at scale, an approach to monitoring that starts with achievable goals and expands in scope as data access improves is likely to be the most appropriate.</p>
<p>The Handbook is presently divided into four broad sections.</p>
<p>Section 1 Biodiversity Informatics. This section focuses on accessing taxonomic data from the Global Biodiversity Information Facility in order to generate species lists for use in monitoring and the use of coordinate data for mapping.</p>
<p>Section 2 Scientific Literature. This section explores methods for obtaining the scientific literature and then combines data on researchers who have held permits with research with the scientific literature.</p>
<p>This section also explores the strengths and weaknesses of the use of ORCID researcher identifier system to retrieve scientific publications linked to researchers who have held permits and the role of identifiers in automating monitoring.</p>
<p>Section 3 Patents. This section will examine the issues involved in monitoring genetic resources and associated traditional knowledge in the patent system. The section will build on existing work and training provided as part of the <a href="https://wipo-analytics.github.io/">WIPO Manual on Open Source Patent Analytics</a>.</p>
<p>Section 4 Data Visualization. Data visualization is a key element in communication and engagement. This section will explore a range of data visualization techniques such as the use of interactive maps and network visualization.</p>
<p>The Handbook will focus on practical methods and tools that countries can use to meet their monitoring needs under the Nagoya Protocol. Much of the work in the Handbook will involve experimentation but is based in the principle of reproducible research. That is, a reader interested in adopting or adapting a particular approach or tool will be provided in the articles with the code and the data that will allow them to do that.</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
